rllib/examples/env/ant_rand_goal.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
```

Comment:
Really cool!
Could you add some comments/docstrings to the methods of these Envs? Or, if these are overrides from parent, could you annotate them using our `@override([]parent)` decorator?
These seem very useful as templates for users to create their custom Env-distributions.

Reply:
Ah they do not override, the Parents dont have sample_tasks, get_task, set_task or step method. However,